### DISCUSSION TITLE
Registration failed, what should I do?  thanks

### DISCUSSION THREAD

**Discussion by declandragon** [huggingface/diffusers#8635] [Q&A] [answered]
![image](https://github.com/huggingface/diffusers/assets/37643710/5291bd90-4825-48a4-976f-8ab8a081dcfe)

**Comment by asomoza** [comment, marked as answer]
Hi, this is the `diffusers` library repository, if you want someone to help you with this problem probably discord would be the best place : https://huggingface.co/join/discord
